client/layout/index.jsx

```diff
@@ -170,6 +173,8 @@ Layout = React.createClass( {
 					{ this.renderWelcome() }
 					{ this.renderEmailVerificationNotice() }
 					<GlobalNotices id="notices" notices={ notices.list } forcePinned={ 'post' === this.props.section.name } />
+					<GuidesPreview showPreview={ this.props.shouldShowGuidesPreview }
```

Comment:
IIRC it was motivated by the possibility of moving away from fixed positioning because of how it interacted with the sidebar and primary. It doesn't matter as much now.